zhaquirks/quirky/tapt.py

```diff
@@ -0,0 +1,187 @@
+"""GE Quirky TAPT device."""
+import logging
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from . import QUIRKY
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+    COMMAND_ON,
+    TURN_ON,
+    TURN_OFF,
+    LONG_PRESS,
+    DIM_UP,
+    DIM_DOWN,
+    LONG_RELEASE,
+    BUTTON,
+    COMMAND,
+    COMMAND_OFF,
+    COMMAND_MOVE,
+    COMMAND_STOP,
+    CLUSTER_ID,
+    ENDPOINT_ID,
+    ARGS,
+)
+
+TAPT_MFG_CODE = 0x5488  # 21640 base 10
+TAPT_NODE_MFG_CODE = 0xBBAA  # 48042 base 10
+TAPT_CLUSTER = 0xFC20  # 64544 base 10
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TaptConfig(CustomCluster):
+    """TAPT Configuration Cluster."""
+
+    cluster_id = TAPT_CLUSTER
+    name = "TAPTCluster"
+    ep_attribute = "tapt_cluster"
+    server_commands = {}
+    client_commands = {}
+    attributes = {
```

Comment:
change this to: `manufacturer_attributes` and then fully remove: `read_attributes_raw` and `write_attributes` below and that should be enough.